packages/twenty-front/src/modules/object-record/record-table/hooks/useCreateNewTableRecords.ts

```diff
@@ -23,7 +23,7 @@ export const useCreateNewTableRecord = (recordTableIdFromProps?: string) => {
 
   const createNewTableRecord = () => {
     setPendingRecordId(v4());
-    setSelectedTableCellEditMode(-1, 0);
+    setSelectedTableCellEditMode('', 0);
```

Comment:
likely what's braking the create behavior. I think we need the pending record section to be on top of the right record group and have the same picker as on board (long term). For now, let's put it to the first recordGroup